tests/legacy-cli/e2e/setup/200-install-cli.ts

```diff
@@ -0,0 +1,16 @@
+import { delimiter, join } from 'path';
+import { getGlobalVariable } from '../utils/env';
+import { npm } from '../utils/process';
+
+// Tested packages required on the $PATH
+const TESTED_BIN_PACKAGES = ['@angular/cli', '@angular-devkit/schematics-cli'];
+
+export default async function () {
+  const testRegistry: string = getGlobalVariable('package-registry');
+
+  // Install the packages being tested
+  await npm('install', `--registry=${testRegistry}`, ...TESTED_BIN_PACKAGES);
```

Comment:
That sounds much better. Then hopefully only the package locations change instead of potential behavioral changes.

I think that will also align more with https://github.com/angular/angular-cli/pull/23074 where the standard global cache and npmrc can't be used because bazel doesn't have permission to access or write to them.